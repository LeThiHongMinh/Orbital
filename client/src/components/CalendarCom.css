.calendar-container {
    background-color: #fff;
    padding: 20px;
    border-radius: 10px;
    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
    margin-bottom: 20px;
}

.calendar-container h2 {
    margin-bottom: 20px;
    font-size: 1.5rem;
    color: #333;
}

.react-calendar {
    border: none;
    width: 100%;
}

.react-calendar__tile--active {
    background: #4caf50 !important;
    color: white;
}

.react-calendar__tile--now {
    background: #ffeb3b !important;
    color: black;
}
