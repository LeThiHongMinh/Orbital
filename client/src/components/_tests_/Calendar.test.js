import * as React from 'react'
import CalendarCom from '../Calendar'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<CalendarCom />)
  expect(tree).toMatchSnapshot()
})