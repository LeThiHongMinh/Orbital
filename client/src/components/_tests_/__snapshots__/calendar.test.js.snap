// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p
    className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  >
    Calendar Component
  </p>
</div>
`;
